src/apps/bridge/network_config_logger.cpp

```diff
@@ -13,7 +13,7 @@ static CborError print_stream_handler(void *out, const char *fmt, ...) {
 
   va_list args;
   va_start(args, fmt);
-  int n = vsnprintf(nullptr, log_buffer_max_size - log_buffer_size, fmt, args);
+  int n = vsnprintf(nullptr, 0, fmt, args);
```

Comment:
This was undefined behavior. It's only valid to pass null if the length is zero.